### DISCUSSION TITLE
bigger tooltips do show scrollbars, but those scrollbars are not keyboard or mouse accessible

### DISCUSSION THREAD

**Discussion by ashutosh-kendurkar** [microsoft/fluentui#25652] [Q&A]
If the tooltip have large text, the scrollbar appears, also works with mouse wheel(scroller) but does not work with keyboard or mouse clicking on the scrollbar.

please see below codepen for the issue.
https://codepen.io/chiragvidani/pen/RwJRNYy

any help is appreciated.

**Comment by ling1726** [comment]
@smhigley  would you know what the correct UI pattern is for handling tooltip content that is scrollable?
